Formula/dynamodb-local.rb

```diff
@@ -67,4 +67,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    system "#{bin}/dynamodb-local", "-help"
```

Comment:
`bin/"dynamodb-local"` instead of `"#{bin}/dynamodb-local"`.

But do you think there's something more substantial than `-help` you can do here?